cmd/plugin/builder/helpers/helper.go

```diff
@@ -85,3 +88,35 @@ func ValidatePluginBinary(pluginBinaryFilePath string) (bool, error) {
 
 	return !fileEmpty, nil
 }
+
+func GetMaxParallelism() int {
+	maxConcurrent := runtime.NumCPU() - 2
+	if maxConcurrent < minConcurrent {
+		maxConcurrent = minConcurrent
+	}
+	return maxConcurrent
+}
+
+var Identifiers = []string{
+	string("=> T01]"),
+	string("=> T02]"),
+	string("=> T03]"),
+	string("=> T04]"),
+	string("=> T05]"),
+	string("=> T06]"),
+	string("=> T07]"),
+	string("=> T08]"),
+	string("=> T09]"),
+	string("=> T10]"),
+	string("=> T11]"),
+	string("=> T12]"),
```

Comment:
Why did we choose to use a different ID than for compilation?
Is it to get rid of non-ascii symbols?

Reply:
That was the initial thinking but I am updated to use the existing symbols instead of updating them in this PR. If needed we can update it in a separate PR.